scripts/appbuilder

```diff
@@ -242,8 +242,13 @@ export_image()
     rm -rf $TEMP_DIR
 }
 
+check_symbols() {
+    $(dirname $0)/symbol_check/part_two_check.sh $IMAGE_NAME "$PWD/$OUTPUT_NAME" $(dirname $0)/symbol_check/symbols/unsupported
+}
+
 get_image
 export_image
+check_symbols
```

Comment:
Can this be a command line opt in option? So users are not surprised by the change in behavior